src/test/java/teammates/test/cases/ui/AllActionsAccessControlTest.java

```diff
@@ -73,7 +73,7 @@ private static void addUnregStudentToCourse1() throws Exception {
     }
 
     @Test
-    public void InstructorCoursesPage() throws Exception {
+    public void instructorCoursesPage() throws Exception {
```

Comment:
Apparently, the `test` prefix is not in fashion anymore since there is a `@Test` annotation.